samples/ControlCatalog/Pages/ClipboardPage.xaml.cs

```diff
@@ -59,24 +65,62 @@ private async void CopyFilesDataObject(object? sender, RoutedEventArgs args)
         {
             if (Application.Current!.Clipboard is { } clipboard)
             {
-                var files = (ClipboardContent.Text ?? String.Empty)
-                .Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
-                if (files.Length == 0)
+                if (TopLevel.GetTopLevel(this)?.PlatformImpl?.TryGetFeature<IStorageProvider>(out var storageProvider) == true)
```

Comment:
Simply use TopLevel.GetTopLevel(this).StorageProvider. 
It also won't be null at this point.